notebooks/official/pipelines/custom_model_training_and_batch_prediction.ipynb

```diff
@@ -0,0 +1,1333 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    import tensorflow as tf
</pre><p>The build is failing on this line. </p><p> </p><p>To resolve it, the following needs to be added in the Installation section:</p><pre class="ql-syntax" spellcheck="false">if os.getenv("IS_TESTING"):
    ! pip install --upgrade tensorflow $USER_FLAG
</pre><p>Example here: https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/master/notebooks/official/explainable_ai/gapic-custom_image_classification_batch_explain.ipynb</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>